test/template-indent.mjs

```diff
@@ -0,0 +1,495 @@
+import stripIndent from 'strip-indent';
+import {getTester} from './utils/test.mjs';
+
+/**
+ * The interesting things to test for this rule are whitespace and multiline templates. Both of those are _very_ hard to see in a
+ * normal text editor, so replace spaces with •, tabs with →→, backticks with @, and $ (template parameter prefix symbol) with #.
+ *
+ * @param {string} text
+ */
+const fixInput = text => stripIndent(text)
+	.replace(/@/g, '`')
+	.replace(/#/g, '$')
+	.replace(/•/g, ' ')
+	.replace(/→→/g, '\t');
```

Comment:
The `•` and `→→` are nice, but `@` and `#` are not very clear when reading the test cases, let's use `` \` `` and `\$` directly?